README.md

```diff
@@ -1 +1,70 @@
-Authenticator Sample App
+# Authenticator sample app
+
+This sample application demonstrates an authenticator-only application that uses PingOne for Customers mobile SDK. An Android developer can easily build a branded and customized Authenticator application using this sample.
+
+## What’s in the sample app?
+
+  - Full native application written in Android, compatible with all Android devices from Android 5 and above.
+  - Integration with PingOne SDK version 1.2.0.
```

Comment:
`Integration with PingOne SDK version 1.2.0.` --> `Integration with PingOne SDK.`
also this should be a link to **master** branch